solution 1.0

aliliolek opened the pull request:
[DEMO LINK](https://aliliolek.github.io/layout_landing-page/)

loralevitska reviewed (changes requested):
You did an amazing job! Looks good! Let's do some improvements (check comments)

- make sure that all interactive elements have a cursor pointer on hover
- implement this text by phone icon hover according design
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/9fa2b0e4-2892-4606-a3d5-20f04633220c)
- add hover effect for this links
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/0f53c9c5-6308-4434-8f67-260e25cf4934)
- remove default autocomplete background color, it breaks design
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/11606096-2fb7-43e8-b188-0188e3d8bd66)
- fix this title on tablet mode according design, also add right line-height 
![image](https://github.com/mate-academy/layout_landing-page/assets/70009205/26ddf580-167e-455c-9f36-101fb12bdd96)

aliliolek commented:
1. Added a **block with the phone numbers** that appears on hover over the call-icon.
2. Added underline for **nav__link:hover**. (as on "call to order")
3. Fixed some issues:
- on **form__field:hover** border-color: #D5D7DE;
- on **form__field:focus** border-color: $main-dark;
However, I don't see in Figma design that **form field`s background-color** should change.

sTorba24 reviewed (changes requested):
Almost good! lets improve it

1. set just `href` attr and use tag `a` - link . But not onclick event
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/afa0da3b-8f08-4ba5-ab86-9d362c7c4efb)

2.still not fixed autocomplete styles for input
![image](https://github.com/mate-academy/layout_landing-page/assets/55200530/58851773-ae37-4815-9e24-5359380ed39b)
